src/ui/React/ThemeEditorModal.tsx

```diff
@@ -96,250 +105,262 @@ export function ThemeEditorModal(props: IProps): React.ReactElement {
-        color={customTheme["int"]}
-        defaultColor={defaultSettings.theme["int"]}
-      />
-      <ColorEditor
-        name="rep"
-        onColorChange={onColorChange}
-        color={customTheme["rep"]}
-        defaultColor={defaultSettings.theme["rep"]}
-      />
-      <ColorEditor
-        name="disabled"
-        onColorChange={onColorChange}
-        color={customTheme["disabled"]}
-        defaultColor={defaultSettings.theme["disabled"]}
-      />
-      <br />
-      <br />
-      <TextField
-        label={"import / export theme"}
-        value={JSON.stringify(customTheme)}
-        onChange={onThemeChange}
-        InputProps={{
-          endAdornment: (
-            <IconButton onClick={resetTheme} size="large">
-              <ReplyIcon />
-            </IconButton>
-          ),
-        }}
-      />
+        <br />
+        <ColorEditor
+          name="hp"
+          onColorChange={onColorChange}
+          color={customTheme["hp"]}
+          defaultColor={defaultSettings.theme["hp"]}
+        />
+        <ColorEditor
+          name="money"
+          onColorChange={onColorChange}
+          color={customTheme["money"]}
+          defaultColor={defaultSettings.theme["money"]}
+        />
+        <ColorEditor
+          name="hack"
+          onColorChange={onColorChange}
+          color={customTheme["hack"]}
+          defaultColor={defaultSettings.theme["hack"]}
+        />
+        <ColorEditor
+          name="combat"
+          onColorChange={onColorChange}
+          color={customTheme["combat"]}
+          defaultColor={defaultSettings.theme["combat"]}
+        />
+        <ColorEditor
+          name="cha"
+          onColorChange={onColorChange}
+          color={customTheme["cha"]}
+          defaultColor={defaultSettings.theme["cha"]}
+        />
+        <ColorEditor
+          name="int"
+          onColorChange={onColorChange}
+          color={customTheme["int"]}
+          defaultColor={defaultSettings.theme["int"]}
+        />
+        <ColorEditor
+          name="rep"
+          onColorChange={onColorChange}
+          color={customTheme["rep"]}
+          defaultColor={defaultSettings.theme["rep"]}
+        />
+        <ColorEditor
+          name="disabled"
+          onColorChange={onColorChange}
+          color={customTheme["disabled"]}
+          defaultColor={defaultSettings.theme["disabled"]}
+        />
+      </Paper>
+
+      <Paper sx={{ px: 1, py: 1, my: 1 }}>
+        <TextField
+          sx={{ mb: 1 }}
+          multiline
+          fullWidth
+          maxRows={3}
+          label={"import / export theme"}
+          value={JSON.stringify(customTheme)}
+          onChange={onThemeChange}
+        />
+          <>
+            <Typography sx={{ my: 1 }}>Copy the string above if you want to backup or share your theme with others.</Typography>
+            <Typography sx={{ my: 1 }}>Use the buttons below to replace the current theme with a pre-built template</Typography>
```

Comment:
Missing dot. Additionally, it's a bit hard to read. see following suggestion:
*Replace the current theme with a pre-built template using the buttons below.*